metrics_api/lib/opentelemetry/metrics/instrument/asynchronous_instrument.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Metrics
+    module Instrument
+      # https://opentelemetry.io/docs/reference/specification/metrics/api/#asynchronous-instrument-api
```

Comment:
This link didn't work for me. I found the doc at:
```suggestion
      # https://opentelemetry.io/docs/specs/otel/metrics/api/#asynchronous-instrument-api
```

Reply:
Good catch! I think they changed it. https://opentelemetry.io/docs/reference/specification/metrics/api/#asynchronous-instrument-api is redirecting to https://opentelemetry.io/docs/specs/otel/metrics/api/#asynchronous-instrument-api with status 301. I'll update them!